src/potions.ts

```diff
@@ -476,10 +479,27 @@ export const rufusPotion = new Potion($item`closed-circuit pay phone`, {
 
     // We consider the average price of the shadow items to not get gated behind an expensive one
     const shadowItems = $items`shadow brick, shadow ice, shadow sinew, shadow glass, shadow stick, shadow skin, shadow flame, shadow fluid, shadow sausage, shadow bread, shadow venom, shadow nectar`;
-    const averagePrice =
-      sum(shadowItems, (it) =>
-        historical && historicalAge(it) < 14 ? historicalPrice(it) : mallPrice(it)
-      ) / shadowItems.length;
+    const getPrice = (item: Item): number =>
+      3 * (historical && historicalAge(item) < 14 ? historicalPrice(item) : mallPrice(item));
+    const avgSpeculativePrice = sum(shadowItems, (it) => getPrice(it)) / shadowItems.length;
+
+    if (haveItemQuest) return avgSpeculativePrice;
+
+    if (get("rufusDesiredItems") === "") {
+      directlyUse($item`closed-circuit pay phone`);
+      runChoice(6);
```

Comment:
Is this the right choice option? 6 feels like it's right for a shadow labyrinth, but I don't think there's any rufus dialog or dialogue with that many options

Reply:
> js availableChoiceOptions()

Returned: aggregate string [int]
1 => I'll fight the entity
2 => I'll find the artifact
3 => I'll collect the goods
6 => Hang up